packages/cli/lib/services/dryrun.service.ts

```diff
@@ -372,6 +379,25 @@ export class DryRunService {
                         break;
                     }
                 }
+
+                if (options.saveResponses) {
+                    const directoryName = `${process.env['NANGO_MOCKS_RESPONSE_DIRECTORY']}${providerConfigKey}`;
+                    if (!fs.existsSync(directoryName)) {
+                        fs.mkdirSync(`${directoryName}/mocks`, { recursive: true });
+                    }
+                    if (!fs.existsSync(`${directoryName}/mocks/${syncName}`)) {
+                        fs.mkdirSync(`${directoryName}/mocks/${syncName}`, { recursive: true });
+                    }
+                    if (nangoProps.rawSaveOutput) {
+                        const filePath = `${directoryName}/mocks/${syncName}/batchSave.json`;
+                        fs.writeFileSync(filePath, JSON.stringify(nangoProps.rawSaveOutput, null, 2));
+                    }
+
+                    if (nangoProps.rawDeleteOutput) {
+                        const filePath = `${directoryName}/mocks/${syncName}/batchDelete.json`;
```

Comment:
not sure if batchDelete is appropriate, there are plenty of time where it's not a batch

Reply:
Will keep as it doesn't harm at all.